src/utils/common.ts

```diff
@@ -7,7 +7,7 @@ export type Decimalish = string | number | Decimal;
 export const AddressZero = ethers.constants.AddressZero;
 export const MAX_BIG_INT = BigNumber.from(Number.MAX_SAFE_INTEGER.toString());
 
-export const { keccak256 } = ethers.utils;
+export const { base58, base64, keccak256 } = ethers.utils;
```

Comment:
@bmzig I see we use `bs58` in `src/utils/AddressUtils.ts`. Is there a specific reason we use that?

(nb. the long term goal is to shuffle off ethers as much as possible, so if there's something better/smaller/whatever then I'm on board with using that. ethers is just convenient because it bundles base64 as well.

Reply:
Yeah you comment about shuffling off of ethers/having a way to work without a dependency on a network-specific library is why. `bs58` is a dependency for other libraries we have (presumably the solana and ethers ones), so we get to add it in our `package.json` without needing to modify `yarn.lock`.

Reply:
Migrated here: https://github.com/across-protocol/sdk/pull/968/commits/bc11c988b29c083a4a3d5031e3a6fd0bdac6aa08